jenkins/aws/buildPython.sh

```diff
@@ -57,8 +57,26 @@ function main() {
     fi
 
     if [[ -f package.json ]]; then
-      npm install --unsafe-perm ||
-        { exit_status=$?; fatal "npm install failed"; return ${exit_status}; }
+      # Select the package manage to use
+        if [[ -z "${NODE_PACKAGE_MANAGER}" ]]; then
```

Comment:
Should we default to npm? or is yarn the preferred package manager at the moment? 
Just worried about backwards compatibility since we use the buildPython task quite a lot